lib/identity/index.js

```diff
@@ -145,17 +145,65 @@ export default class Identity extends CryptoConsumer {
       }
     )
     /* eslint-enable */
-    /* eslint-disable-next-line no-warning-comments */
-    // TODO: Write recovery note based on final parameters
+    const brokerClientID = '00000000-0000-0000-0000-000000000000'
```

Comment:
Do we want this to broker flow to be optional//higher security tradeoffs?

Reply:
possibly -- but we don't fully understand the various use-cases right now to set up options correctly, so I think hard coded for our one use case is the way to go at this time, then we can make adjustments as it grows.

Reply:
Agree, but would still prefer to see the broker flow be gated behind a Boolean parameter for this method that we can default to true, but thats a change we could just as easily make in the future 🤷🏻‍♀️

Reply:
I started adding this, but I don't like any way it starts looking. I'm going to leave this hard-coded right now, and when we start doing our first alternate broker-based, or removing broker-based authentication we can refactor to support it.